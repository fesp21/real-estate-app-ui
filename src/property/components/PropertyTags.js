import React, { Component, PropTypes } from 'react';
import { View, StyleSheet, Text } from 'react-native';

export default class PropertyTags extends Component {

  static propTypes = {
    tags: PropTypes.array.isRequired
  };

  render() {
    const {tags} = this.props;
    return (
      <View style={styles.container}>
        {
          tags.map((tag,i)=>
            <Text key={i} style={styles.tag}> #{tag} </Text>
          )
        }
      </View>
    );
  }

}

const styles =  StyleSheet.create({
  container : {
    flexDirection:'row',
  },
  tag:{
    color:'gray',
    fontSize:11
  }
});