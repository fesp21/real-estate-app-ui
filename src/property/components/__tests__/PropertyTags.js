import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import PropertyTags from "./../PropertyTags";

test('renders heart', () => {
  const tree = renderer.create(
    <PropertyTags
      tags={['A','B','C']}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});