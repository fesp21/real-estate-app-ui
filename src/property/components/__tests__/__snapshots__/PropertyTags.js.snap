exports[`test renders heart 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }>
     #
    A
     
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }>
     #
    B
     
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "gray",
        "fontSize": 11,
      }
    }>
     #
    C
     
  </Text>
</View>
`;
